src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -498,36 +504,36 @@ def _upload_assembly_files_in_parallel(
         workspace_id: target workspace id
         load_id: load id
         upload_dir: a directory in collectionssource that creates new directories linking to sourcedata
-        wait_to_upload_assemblies: a dictionary that maps assembly file name to assembly directory
+        wait_to_upload_objs: a dictionary that maps object(assembly) file name to object(assembly) directory
```

Comment:
Why the parentheses here but no where else?

Reply:
it's in `_upload_assembly_files_in_parallel` methods. It's specific to Assembly.

Reply:
it's in `_upload_assembly_files_in_parallel` methods. It's specific to Assembly.

Reply:
:+1: